docs/getting-started.md

```diff
@@ -130,7 +130,7 @@ Now it's time to write the image to the card.
 <div class="alert alert-info" markdown="1">
 {% include icon.html type="info" %}
 For more detailed information and more alternatives, check out
-[our other tutorials](/docs/tutorials/#group-sd-card-image){: .alert-link }.
+[our other tutorials](/docs/tutorials/#group-administration-and-setup-sd-card-image){: .alert-link }.
```

Comment:
The ID is composed of the group name and the subgroup name, so this is correct for the current behavior. The CI server checks complained at me after I changed the tutorial page because I had forgotten to update this link, but now it's happy that the link is valid.